packages/engine/tests/simulations/message_sending/experiments.json

```diff
@@ -0,0 +1,10 @@
+{
+  "test": {
```

Comment:
I think it'd be better to have clearer names like "send-one" or something, so the resultant outputs end up a bit like
> `message_sending/send_one/1`
rather than 
> `message_sending/test/1`

And by resultant outputs I just mean in various places like the outputs folder etc. not necessarily the results of the actual testing apparatus